app/javascript/dashboard/routes/dashboard/settings/inbox/components/EditCampaign.vue

```diff
@@ -209,3 +213,25 @@ export default {
   },
 };
 </script>
+<style lang="scss" scoped>
+.editor-wrap {
+  margin-bottom: var(--space-normal);
+}
+
+.editor {
+  border: 1px solid var(--color-border);
+  border-radius: var(--border-radius-normal);
+  padding: 0 var(--space-slab);
+  margin-bottom: 0;
+}
+
+.editor_warning {
+  border: 1px solid var(--r-400);
+}
+
+.warning-message {
+  color: var(--r-400);
+  font-weight: var(--font-weight-normal);
+  padding: var(--space-smaller) 0 0 0;
+}
```

Comment:
This seems to be repeated, please explore having this style in the editor component itself.